docs/reference/release-notes/index.md

```diff
@@ -1,11 +1,14 @@
 (reference-release-notes-index)=
-# Release-Notes
+# Releases
+
+Charmed Apache Kafka K8s has the following stable revisions:
+
+<!-- Add additional information regarding version compatibility of different revisions here -->
 
 ```{toctree}
 :titlesonly:
 :maxdepth: 2
-:glob:
-:hidden:
 
-*
+82<revision-82.md>
+56/51<revision-56-51.md>
```

Comment:
Do we need these on the 4 track? I'm not sure how they apply here

Reply:
We've discussed it with Marc O. some months ago. They are here mostly to keep it from being empty. I see no harm. I'd remove them later, when we have some selection of revision for the v.4.